src/lib/core/components/visualizations/implementations/BoxplotVisualization.tsx

```diff
@@ -463,11 +463,7 @@ function BoxplotViz(props: VisualizationProps) {
       }
       orientation={'vertical'}
       // add condition to show legend when overlayVariable is used
```

Comment:
I guess we can remove this comment now!

Reply:
yep I removed it 👍 Also removed unnecessary console.logs that were found at the same component 😄